.github/workflows/prometheus-exporter-helm-release.yaml

```diff
@@ -17,13 +12,18 @@ jobs:
         with:
           fetch-depth: 0
 
+      # We set the chart release version here - the version schema
+      # is a SemVer adherent date-based versioning scheme that looks like:
+      # 2024.2.9125019
+      # which equates to a release on 2/9/24 at 12:50:19
       - name: Get the version tags
         id: get_version
         run: |
           # Enable pipefail so git command failures do not result in null versions downstream
           set -x
-          export "FULL_RELEASE=$(echo $GITHUB_REF | cut -d / -f 3)"
-          echo "RELEASE_VERSION=$(echo $FULL_RELEASE | cut -d - -f 4)" >> $GITHUB_OUTPUT
+
+          echo "CHART_VERSION=$(date +'%Y.%-m.%-d%H%M%S')" >> $GITHUB_OUTPUT
```

Comment:
chart version: 2024.15.5xxxxxx
release version: prefect-prometheus-exporter-2024.15.5xxxxxx

Reply:
release is just the github release/tag, right

Reply:
wdym